### PR TITLE
move to 3scale GA operator

### PR DESCRIPTION
**Pull request by philbrookes**
- update integreatly to 1.8.2 replacing 1.8.0
- update 3scale to 1.9-7 (1.9.7 where format is required) with no replace.

### CODE REVIEW

File: integreatly/integreatly-1.8.2/integreatly-operator.1.8.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+    createdAt: 2019-04-30 12:39:00
+    support: Integreatly
+    description: Greatly integrating integreatly greatness
+spec:
+  displayName: Integreatly
+  description: Integreatly installation and reconciliation operator
+  keywords: ['integreatly']
+  version: 1.8.2
+  maturity: alpha
+  replaces: integreatlyoperator.1.8.0
```
Comment (austincunningham) [inline integreatly/integreatly-1.8.2/integreatly-operator.1.8.0.clusterserviceversion.yaml]: No 1.8.1 version I guess

File: integreatly/integreatly-1.8.2/integreatly-operator.1.8.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,366 @@
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (austincunningham) [inline integreatly/integreatly-1.8.2/integreatly-operator.1.8.0.clusterserviceversion.yaml]: need to update the file name :)

File: integreatly/integreatly-1.8.2/integreatly-operator.1.8.0.clusterserviceversion.yaml (hunk 1 above)
Comment (philbrookes) [reply integreatly/integreatly-1.8.2/integreatly-operator.1.8.0.clusterserviceversion.yaml]: doesn't seem that way!

File: integreatly/integreatly-1.8.2/integreatly-operator.1.8.0.clusterserviceversion.yaml (hunk 2 above)
Comment (philbrookes) [reply integreatly/integreatly-1.8.2/integreatly-operator.1.8.0.clusterserviceversion.yaml]: good spot! done.

**Pull request merged**